code/poseidonperm_x5_vesta_3.sage

```diff
@@ -75,9 +75,9 @@ round_constants = [
 ]
 
 MDS_matrix = [
-    [0x0ab5e5b874a68de7b3d59fbdc8c9ead497d7a0ab23850b56323f2486d7e11b63, 0x31916628e58a5abb293f0f0d886c7954240d4a7cbf7357368eca5596e996ab5e, 0x07c045d5f5e9e5a6d803952bbb364fdfa0a3b71a5fb1573519d1cf25d8e8345d],
-    [0x233162630ebf9ed7f8e24f66822c2d9f3a0a464048bd770ad049cdc8d085167c, 0x25cae2599892a8b0b36664548d60957d78f8365c85bbab07402270113e047a2e, 0x22f5b5e1e6081c9774938717989a19579aad3d8262efd83ff84d806f685f747a],
-    [0x2e29dd59c64b1037f333aa91c383346421680eabc56bc15dfee7a9944f84dbe4, 0x1d1aab4ec1cd678892d15e7dceef1665cbeaf48b3a0624c3c771effa43263664, 0x3bf763086a18936451e0cbead65516b975872c39b59a31f615639415f6e85ef1],
+    [0x1853b4977c6fa22791913f56cf21af2b5f710afc43ddc5f6eb4f1f742963421f, 0x3d831189cfbbc45263f484c10fcf05865a0fa4dfa500bcad45e51db6ac6fe4a7, 0x3a0e3f84d3c177d84ba88b9e401719c03f8965c780838a94d18837f98347f137],
+    [0x35e26e39845062798e9dc529f4718f832896f8d0fd5c9a7584fd7923337cf77e, 0x10a8166302cb753c00cd7dbea79970ab3641cecf3a2a5a8a3eb924f56fff7908, 0x314f762a506d321bf43492ce51214b00198e1aee777e2521b67227c1a141ae94],
+    [0x07b85627c832782acb2eab86ef31d915a90f28b0cb3176fbabcbd614eaf5eba1, 0x2a2de13e70f27e1697564e1b5d1ac72fb5d985dc1630a4b2c255efd006b5db1c, 0x2c6094d1c6e1cabafff540a87327c7ce21e3af7ef12332cdcffdf529333429fc],
```

Comment:
I checked these. Not sure why they didn't get updated before; probably a cut-and-paste error.